internal/config/config_test.go

```diff
@@ -420,6 +420,11 @@ func TestLoad(t *testing.T) {
 				return cfg
 			},
 		},
+		{
+			name:    "authentication failure on OIDC email matching",
+			path:    "./testdata/authentication/oidc_email_invalid_matching.yml",
+			wantErr: errFieldWrap("authentication.methodoidc.oidc.email_matches", fmt.Errorf("email match string: ) invalid")),
+		},
```

Comment:
The environment variables is a bit weird when it comes to Viper binding array values.

It looks like:
```
FLIPT_AUTHENTICATION_METHODS_OIDC_EMAIL_MATCHES=[')']
```

translates to `[[')']]` in Flipt. Does anybody know why?